kwanj-k/storemanager-API
# #161199192 Add a make sale endpoint

### What does this PR do?
Add a make a sale endpoint

#### Description of Task to be Completed ?
-- Have a working make sale endpoint
-- Consider different edge cases when making a sale

### How should this be manually tested?
* After cloning the repo,git pull the branch ft-make-sale-#161199192
* Install a virtual enviroment and activate it [Refer here](https://docs.python-guide.org/dev/virtualenvs/)
* Pip install the the requirements.txt
* Run the following:
  ``` export FLASK_APP='run.py' ```
  ``` export APP_SETTINGS='development' ```
   ``` pytest --cov-report term-missing --cov=app ```

 All related products tests should pass

### What are the relevant Pivotal Tracker stories?
  #161199192
